### DISCUSSION TITLE
Question about the api_cache

### DISCUSSION THREAD

**Discussion by bm55** [photoview/photoview#405] [Q&A and Troubleshooting]
I noticed that there are 2 images:
thumbnail... and highres...
Does Photoview use the highresxxx.jpg to display the image? I am curios if the highresxxx.jpg can be skipped for smaller size photos and use the original photo to display on the Photoview. This will keep the cache size smaller. I got a lot of cell phone photos that are around 2MB and the cache creates another 1.2MB highresxxx.jpg

**Comment by viktorstrate** [comment]
Highres images should only be generated for images where the original image is not directly web compatiable such as RAW and TIFF images. If the original image is a JPEG or PNG it will use the original instead of generating a new highres version.

**Reply by bm55** [reply]
Thanks for clarifying and replying so quickly!
I am assuming heic is not web compatible. It looks like most of the original files are iPhone's heic files.
